models/example/fct_order_items.sql

```diff
@@ -0,0 +1,30 @@
+{# 
+create a `fct_order_items` table joining orders to customers to get the 
+`order_date` and `email` and `customer_id` on the order table
+#}
+with source_1 as (
+
+    select * from {{ ref('orders_upload') }}
+
+),
+
+source_2 as (
+
```

Comment:
this task was written super unclear. I just updated it in the template so it isn't as confusing moving forward.

what it should have said is 

"create a `fct_order_items` table joining orders to order items. select all fields from order items and join in the `order_date`, `email` and `customer_id` from the orders table"

can you update your code with those parameters?